src/popup/locales/en.json

```diff
@@ -127,12 +127,13 @@
       "consensusRequiredHelpTitle": "Consensus required",
       "consensusRequiredHelpMsg": "As the creator of this multisig vault you can determine the authorized signers and set the consensus needed for a transaction confirmation (how many signatures are required to confirm each transaction).",
       "createMultisigVault": "Create Vault",
-      "errorDuplicatingSigner": "Cannot add the same address as a signer twice"
+      "errorDuplicatingSigner": "Cannot add the same address as a signer twice",
+      "errorNoEnoughBalanceToCreateVault": "No enough balance to create multisig vault."
```

Comment:
We can use on of the existing `not enough balance` wordings
https://github.com/aeternity/superhero-wallet/blob/b5dccbc6c706b7be44298cc5d566275fac2c3214/src/popup/locales/en.json#L747
https://github.com/aeternity/superhero-wallet/blob/b5dccbc6c706b7be44298cc5d566275fac2c3214/src/popup/locales/en.json#L1043
https://github.com/aeternity/superhero-wallet/blob/b5dccbc6c706b7be44298cc5d566275fac2c3214/src/popup/locales/en.json#L1414

or at least adjust the message to be "No**t** enough..."